modules/ROOT/pages/logging/log-downloads.adoc

```diff
@@ -11,43 +11,48 @@
 label:AuraDB-Professional[]
 label:AuraDB-Virtual-Dedicated-Cloud[]
 
-Both *Query log analyzer* and *Security log analyzer* allow you to download the fetched logs.
-The downloaded logs account for any filters and search that have been applied.
+Downloading logs requires a project role of either _{role-project-admin}_ or _{role-project-member}_.
+
+You can download logs from the *Query log* and the *Security log*.
+The logs include any filters or search terms applied when the logs were fetched.
 Downloading logs consists of two steps: **initiate the download** and **download the file**.
 
-[NOTE]
-====
-Downloading logs requires a project role of either _{role-project-admin}_ or _{role-project-member}_.
-====
+== Initiate logs 
 
-== Initiate download
+Each request is up to {max-request-hours} hours of data and a maximum of {max-download-rows} rows.
 
-To initiate a download, after fetching the logs, use the _Initiate log download_ icon button (Arrow Down Tray Icon) in the top right corner of either the Summary or Details table.
-Select the following options in the dialog:
+After fetching logs, use the *Initiate logs download* button (arrow down tray icon) in the top right corner of the *Summary* and *Details* table. 
 
-* *Log type* - Select one or both:
-** *Summary* - Aggregated log statistics
-** *Details* - Individual log events
-* *Format* - Choose your preferred format:
-** *JSON*
-** *CSV* with optional settings:
-*** Include CSV headers (default: enabled)
-*** CSV field delimiter (default: comma)
+* Select the *log type*: Summary (aggregated log statistics), Details (individual log events), or both. 
+* Choose a format: JSON or CSV. 
+If you choose CSV, you can optionally include CSV headers (enabled by default) and specify a field delimiter (default: comma).
+* Select *Confirm* to request the logs.
 
-*Confirm* starts the download.
+When the requested log is ready, it appears in the *Downloads archive.*
```

Comment:
This is a bit weird since the log isn't actually downloaded at this point. But that's a UI problem.